rasa/core/channels/socketio.py

```diff
@@ -155,6 +156,7 @@ def __init__(
         socketio_path: Optional[Text] = "/socket.io",
         jwt_key: Optional[Text] = None,
         jwt_method: Optional[Text] = "HS256",
+        metadata_key: Optional[Text] = "metadata",
```

Comment:
If we are passing the value that corresponds to `metadata_key` in `credentials` dict, then that can be of type `Optional[Any]`? If it must be `Optional[Text]` the docs need to add this clarification, because the example uses `customData` and that threw me off a bit 😅

Reply:
I don't understand. The `metadata_key` value would need to be text.

Reply:
I got confused by the docs yaml example `metadata_key: customData` and expected the value in this pair to be the actual metadata object or dictionary. But I think you actually want this to be a string key, so I'd amend the example in the docs with `metadata_key: your_custom_metadata_key`

Reply:
Thanks, I chose `customData` because the most widely used rasa chat widget, [botfront widget](https://github.com/botfront/rasa-webchat#rasa) stores metadata under this key.